src/compas_model/model/model.py

```diff
@@ -117,7 +116,7 @@ def __getitem__(self, index):
     # Info
     # ==========================================================================
 
-    def print(self):
+    def print_model(self):
```

Comment:
perhaps we should just switch to using `__str__` and print with `print(model)`

Reply:
sounds good to me but would have to make changes in `Tree.print_hierarchy` and `InteractionGraph.print_interactions` as they internally call `print`

Reply:
btw I think generally we should avoid `print` calls in COMPAS, especially the parts that are more library-ish.

Reply:
Let's change to `__str__` and then additional change will be needed for the `tree `in compas and merge this pull request.
As we discussed, still ironpython needs to be supported for some time.